### PR TITLE
Add support for embedding soundcloud oembed

### PR DESCRIPTION
**Pull request by christofdamian**
I have to confess that I haven't tested if this works yet. 
Adding another embed involves quite a few changes to files. 
This is for DVX-7041

### CODE REVIEW

File: spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'embed_parser_shared'
+
+describe ArticleJSON::Import::GoogleDoc::HTML::EmbeddedSoundcloudParser do
+  include_context 'for an embed parser' do
+    let(:expected_embed_type) { :soundcloud }
+    let(:expected_embed_id) { 'rich-the-kid/plug-walk-1' }
+    let(:expected_tags) { %w(vimeo test) }
```
Comment (dsager) [inline spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb]: `s/vimeo/soundcloud/`?

Comment (christofdamian) [reply spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb]: good point,  or just test? what is it for anyway?

Comment (dsager) [reply spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb]: Acutally the values don't really matter: the shared example writes the tags into the HTML fixture and then expects them to be extracted properly. Might still be less confusing to not use `vimeo` here ;)

**Comment by bbrassart** [comment]
+1 I think

File: spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb (hunk 1 above)
Comment (christofdamian) [reply spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb]: OK, I am going to do that. I guess I also have to add the support for AMP.

Comment (dsager) [reply spec/article_json/import/google_doc/html/embedded_soundcloud_parser_spec.rb]: I'm not entirely sure, but I think AMP should work out of the box

**Comment by christofdamian** [comment]
@dsager I fixed the tags and added an amp exporter

**Review by dsager — APPROVED** [review]
:+1:

**Comment by christofdamian** [comment]
codeclimate is correct, but it isn't my fault!

**Comment by dsager** [comment]
I approved the warnings :)

**Pull request merged**